package.json

```diff
@@ -123,10 +124,11 @@
     "webpack-bundle-size-analyzer": "^2.0.2",
     "webpack-bundle-tracker": "https://github.com/learningequality/webpack-bundle-tracker",
     "webpack-dev-middleware": "^1.9.0",
-    "webpack-dev-server": "^2.3.0",
-    "webpack-hot-middleware": "^2.12.1",
-    "webpack-merge": "^4.1.0",
-    "webpack-rtl-plugin": "^1.6.0"
+    "webpack-dev-server": "2.3.0",
+    "webpack-hot-middleware": "2.12.1",
+    "webpack-merge": "^2.5.0",
```

Comment:
Can you revert the change to the `webpack-merge` version here? It should be `^4.1.0`. Will need to regenerate the yarn.lock file when you do this also.